bots/guess_bot.test.js

```diff
@@ -0,0 +1,28 @@
+const { GuessBot } = require('./guess_bot')
+
+jest.mock('../connection')
+
+class TestBot extends GuessBot {
+  handleReward = jest.fn()
+  handleExplorationSpace = jest.fn()
+}
+
+test('Handles reward message properly', () => {
+  const text = `<@TESTID> You got a response reward 10, jump, 90, 100`
+  const bot = new TestBot()
+
+  bot._handleMessage({ text })
+  expect(bot.handleReward.mock.calls[0][0]).toBe('10');
+  expect(bot.handleReward.mock.calls[0][1]).toBe('jump');
+  expect(bot.handleReward.mock.calls[0][2]).toBe('90');
+  expect(bot.handleReward.mock.calls[0][3]).toBe('100');
```

Comment:
Instead of four `expect`s here, what do you think about replacing them with `expect(bot.handleReward).toBeCalledWith('10', 'jump', '90', '100');`?

Reply:
Addressed in 30759a555c20f04ab6f7fc2235101bfdce3408c6